### PR TITLE
Dashboard Visual Style Revamp (2020Mar)

### LINKED ISSUE #862: [Demand] Dashboard Visual Style Revamp (2020Mar)
## Metadata

Field | Value
--- | ---
Previous Stage | ...
Figma Link | https://www.figma.com/file/4DmpRxZk4jDeqrfA5xrPCp/Maskbook-Dashboard?node-id=1%3A168
Expected UX prototype due | 2020-W??-5
Actual UX prototype release | -
Expected engineering due | 2020-W??-5
Corresponding Pull Request | -
Completion Version | -

## Catalogue

- Home
- Personas
- Wallets

## Notes

- This is a draft. More contents will be added when Figma prototypes are ready for review.

### PR DESCRIPTION
**Pull request by guanbinrui**
Rebase #902 onto the `master`.

Close #862

### CODE REVIEW

**Comment by Jack-Works** [comment]
@guanbinrui I have rebased this branch

**Comment by guanbinrui** [comment]
> @guanbinrui I have rebased this branch

OK

**Comment by Jack-Works** [comment]
@guanbinrui rebased

File: src/_locales/zh/messages.json [hunk 1]
```diff
@@ -142,6 +142,7 @@
     "go_to": "前往",
     "history": "歷史",
     "home": "主頁",
+    "error_meme_unmatch": "",
```
Comment (Jack-Works) [inline src/_locales/zh/messages.json:145]: Why an empty string?

Comment (guanbinrui) [reply src/_locales/zh/messages.json:145]: ahh... Should be removed.

File: src/extension/options-page/DashboardRouters/Settings.tsx [hunk 2]
```diff
@@ -71,25 +76,6 @@ const settingsTheme = (theme: Theme): Theme =>
...
-                root: {
-                    paddingTop: theme.spacing(1.5),
-                    paddingBottom: theme.spacing(1.5),
-                    borderBottom: `1px solid ${theme.palette.divider}`,
-                },
-                secondaryAction: {
-                    paddingRight: '90px',
-                },
-            },
-            MuiListItemIcon: {
```
Comment (guanbinrui) [inline src/extension/options-page/DashboardRouters/Settings.tsx]: Why delete this?

File: src/extension/options-page/DashboardComponents/ContactLine.tsx [hunk 3]
```diff
@@ -30,7 +30,7 @@ const useStyles = makeStyles((theme) =>
             whiteSpace: 'nowrap',
         },
         provider: {
-            color: 'var(--lightText)',
+            color: theme.palette.text.disabled,
```
Comment (guanbinrui) [inline src/extension/options-page/DashboardComponents/ContactLine.tsx:33]: Better use `secondary`.

File: src/extension/options-page/index.tsx [hunk 4]
```diff
@@ -45,7 +45,7 @@ const useStyles = makeStyles((theme) => {
         wrapper: {
             '--monospace': 'SFMono-Regular, Consolas, "Liberation Mono", Menlo, monospace',
 
-            '--drawerHeader': dark ? '#121212' : theme.palette.primary.dark,
+            '--drawerHeader': dark ? '#121212' : theme.palette.primary.main,
```
Comment (Jack-Works) [inline src/extension/options-page/index.tsx:48]: Hmm why

Comment (guanbinrui) [reply src/extension/options-page/index.tsx:48]: It's too dark as the background color of the drawer.

**Comment by Tedko** [comment]
Screen Shot 2020-05-25 at 17 47 46



@guanbinrui bug to fix

**Comment by guanbinrui** [comment]
> Screen Shot 2020-05-25 at 17 47 46
> 
> @guanbinrui bug to fix

Fixed.

**Comment by guanbinrui** [comment]
> Screen Shot 2020-05-25 at 17 47 46
> 
> @guanbinrui bug to fix

Fixed by https://github.com/DimensionDev/Maskbook/pull/1026/commits/32b3943e6158cc968febba36396515b1a5ffe5c0.

**Comment by Tedko** [comment]
> > Screen Shot 2020-05-25 at 17 47 46
> > @guanbinrui bug to fix
> 
> Fixed.

ok/ so is the test net toggle?

**Comment by Tedko** [comment]
Also, 

for the default Token list, make it:

- DAI (since its stable coin)
- ETH (you need it as gas)

File: src/plugins/Wallet/UI/Developer/EthereumNetworkSettings.ts [hunk 5]
```diff
@@ -43,9 +42,9 @@ export function getNetworkSettings() {
 
 export const currentEthereumNetworkSettings = createNewSettings<EthereumNetwork>(
     'eth network',
-    EthereumNetwork.Mainnet,
+    EthereumNetwork.Rinkeby,
```
Comment (Jack-Works) [inline src/plugins/Wallet/UI/Developer/EthereumNetworkSettings.ts:45]: Default to Mainnet?

Comment (guanbinrui) [inline src/plugins/Wallet/UI/Developer/EthereumNetworkSettings.ts:45]:
```suggestion
    EthereumNetwork.Mainnet,
```

File: src/_locales/en/messages.json [hunk 6]
```diff
@@ -105,88 +128,121 @@
...
     "ok": "OK",
     "options_index_dev": "Developer Options",
     "options_index_mobile_export": "Sync with mobile",
     "options_index_privacy": "Privacy Policy",
     "options_index_setup": "Setup your account",
-    "options_mobile_export_generating": "Generating QR Code...",
+    "options_mobile_export_generating": "Generating QR Code…",
     "options_mobile_export_subtitle": "DO NOT share it with others!",
+    "password": "Password",
+    "extra_password": "Extra Password",
```
Comment (Jack-Works) [inline src/_locales/en/messages.json:193]: What is this "extra password"?

File: src/_locales/zh/messages.json [hunk 7]
```diff
@@ -1,34 +1,54 @@
 {
+    "about": "關於",
+    "about_dialog_description": "Maskbook 允許您在社交網絡上發布加密的帖子，從而可以減輕大公司對大數據的監視。只有您和您的朋友（也在使用 Maskbook）可以解密您的加密帖子。",
+    "about_dialog_touch": "保持聯繫",
```
Comment (Jack-Works) [inline src/_locales/zh/messages.json:4]: `聯係我們`?

File: src/_locales/zh/messages.json [hunk 8]
```diff
@@ -1,34 +1,54 @@
...
     "automation_request_paste_into_bio_box": "請拷貝公鑰，然後發佈到個人簡介上。",
     "automation_request_paste_into_post_box": "請拷貝公鑰，然後發佈到時間綫上。",
+    "new_avatar_url": "新的頭像網址",
+    "new_avatar_url_placeholder": "無需修改時請留空",
     "back": "返回",
     "backup": "備份",
+    "backup_database": "備份資料庫",
     "backup_persona": "備份角色",
+    "backup_wallet": "備份錢包",
+    "backup_wallet_hint": "請<strong>小心</strong>將以下<strong>12個助記詞</strong>或<strong>私鑰</strong>放在安全的地方，您需要它們來還原此錢包。",
```
Comment (Jack-Works) [inline src/_locales/zh/messages.json:33]:
```suggestion
    "backup_wallet_hint": "請<strong>務必</strong>將以下<strong>12個助記詞</strong>或<strong>私鑰</strong>放在安全的地方，您需要它們來還原此錢包。",
```

File: src/extension/options-page/DashboardRouters/Setup.tsx [hunk 9]
```diff
@@ -0,0 +1,879 @@
...
+            marginBottom: theme.spacing(2),
+        },
+        secondary: {
+            textAlign: 'center',
+            fontSize: 20,
+            lineHeight: 1.5,
+            marginBottom: theme.spacing(5),
+        },
+        form: {
+            width: 368,
```
Comment (Jack-Works) [inline src/extension/options-page/DashboardRouters/Setup.tsx:73]: not suggest hard coding width

File: src/extension/options-page/Dialogs/Feedback.tsx [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+                            rows={4}
+                            required
+                            label={t('your_message')}
+                            value={message}
+                            onChange={(e) => setMessage(e.target.value)}
+                        />
+                    </form>
+                }
+                footer={
+                    <DebounceButton onClick={async () => {}} variant="contained" color="primary">
```
Comment (Jack-Works) [inline src/extension/options-page/Dialogs/Feedback.tsx:34]: Not implemented?

File: src/_locales/zh/messages.json (hunk 8 above)
Comment (guanbinrui) [reply src/_locales/zh/messages.json:33]: chore: update i18n translation

File: src/_locales/en/messages.json (hunk 6 above)
Comment (guanbinrui) [reply src/_locales/en/messages.json:193]: Removed

File: src/extension/options-page/Dialogs/Feedback.tsx (hunk 10 above)
Comment (guanbinrui) [reply src/extension/options-page/Dialogs/Feedback.tsx:34]: @yisiliu @septs The feedback API is ready now? Or we'd like just redirect our users to the Github issue page?

File: src/components/InjectedComponents/ImmersiveGuide/SetupGuide.tsx [hunk 11]
```diff
@@ -0,0 +1,475 @@
...
+    })
+
+const useWizardDialogStyles = makeStyles((theme) =>
+    createStyles({
+        root: {
+            boxSizing: 'border-box',
+            padding: '48px 20px 0',
+            position: 'relative',
+            width: 320,
+            minHeight: 404,
```
[the remark this answers is not in the document]
Comment (guanbinrui) [reply src/components/InjectedComponents/ImmersiveGuide/SetupGuide.tsx:98]: The Mobile is not supported.

File: src/extension/options-page/DashboardRouters/Setup.tsx (hunk 9 above)
Comment (guanbinrui) [reply src/extension/options-page/DashboardRouters/Setup.tsx:73]: The Mobile is not supported.

**Pull request merged**